bot/cogs/cyber.py

```diff
@@ -211,63 +212,40 @@ def __init__(self, bot: Bot):
 
     @command()
     async def game(self, ctx: Context):
-        # Get the current date
-        today = datetime.date.today()
-        game_start_date = datetime.date(2019, 1, 15)
-        time_until_game = relativedelta(game_start_date, today)
-
-        # Given a number of items, determine whether it should be pluralised.
-        # Then, return the suffix of 's' if it should be, and '' if it shouldn't.
-        def suffix_from_number(num):
-            return "" if num == 1 else "s"
-
-        month_or_months = "month" + suffix_from_number(time_until_game.months)
-        day_or_days = "day" + suffix_from_number(time_until_game.days)
-
-        month_countdown = f"{time_until_game.months} {month_or_months}"
-        day_countdown = f"{time_until_game.days} {day_or_days}"
-
-        # Diable the months component of the countdown when there are no months left
-        if time_until_game.months:
-            month_and_day_countdown = f"{month_countdown} and {day_countdown}"
-        else:
-            month_and_day_countdown = day_countdown
-
-        if today > game_start_date:
-            await ctx.send("Cyberstart Game has begun! Use :level base level to get info"
-                           "on specific challenges once we update the bot")
-            return
-        await ctx.send("Cyberstart Game begins on the 15th January 2019.")
-        await ctx.send(f"That's in {month_and_day_countdown}!")
+        await self.countdown('15th January 2019', 'CyberStart Game', ctx)
 
     @command()
     async def essentials(self, ctx: Context):
```

Comment:
Add a docstring explaining the command. The `:help` function uses this docstring to generate a description, see my screenshot above.